Add the Gentoo lib64 cmake path to the system flatbuffers search list and
add missing unversioned target aliases that npu_compiler's cmake scripts
expect. (The ENABLE_INTEL_NPU dependency for ENABLE_SYSTEM_FLATBUFFERS in
cmake/features.cmake is upstreamed in this OpenVINO revision.)

--- a/thirdparty/dependencies.cmake	2026-09-13 01:42:52.341229788 -0700
+++ b/thirdparty/dependencies.cmake	2026-09-13 01:42:52.341275818 -0700
@@ -447,7 +447,8 @@
 
         # on new Ubuntu versions like 23.04 we have config called FlatBuffersConfig.cmake
         # so, we need to provide alternative names
-        find_host_package(Flatbuffers QUIET NAMES Flatbuffers FlatBuffers NO_CMAKE_FIND_ROOT_PATH)
+        find_host_package(Flatbuffers QUIET NAMES flatbuffers Flatbuffers FlatBuffers NO_CMAKE_FIND_ROOT_PATH
+            PATHS /usr/lib64/cmake/flatbuffers)
 
         ov_cross_compile_define_debian_arch_reset()
     endif()
@@ -462,6 +463,12 @@
             message(FATAL_ERROR "Internal error: Failed to detect flatbuffers library target")
         endif()
         set(flatbuffers_COMPILER flatbuffers::flatc)
+        if(NOT TARGET flatbuffers)
+            add_library(flatbuffers ALIAS ${flatbuffers_LIBRARY})
+        endif()
+        if(NOT TARGET flatc)
+            add_executable(flatc ALIAS flatbuffers::flatc)
+        endif()
     else()
         add_subdirectory(thirdparty/flatbuffers EXCLUDE_FROM_ALL)
         if(ENABLE_INTEL_NPU)
